data/test/scenarios/README.md

```diff
@@ -49,5 +54,9 @@ existing tests don't conform to this goal yet.
 Names containing `_fail_` or ending `_fail` are reserved for tests that should not `PASS`. However,
 they may expect a status that is neither `PASS` nor `FAIL`.
 
-Names containing `break` or `error` might be for tests expected to `PASS`. Some of these are testing
-loops, or testing error-handling that is expected to handle the error.
+Names containing `_bug_` or ending `_bug` are reserved for tests where consensus is that the
```

Comment:
ending in?

Reply:
With or without both work in the UK, and adding "in" would need to be done to the `_fail` paragraph too.